ui/app/adapters/mfa-method.js

```diff
@@ -3,24 +3,26 @@ import ApplicationAdapter from './application';
 export default class MfaMethodAdapter extends ApplicationAdapter {
   namespace = 'v1';
 
-  urlForQuery(methodType) {
-    let baseUrl = this.buildURL() + '/identity/mfa/method';
-    if (methodType) {
-      return `${baseUrl}/${methodType}`;
-    }
-    return baseUrl;
+  pathForType() {
+    return 'identity/mfa/method';
   }
 
-  queryRecord(type, id) {
-    return this.ajax(this.urlForQuery(type), 'POST', {
+  queryRecord(store, type, query) {
+    const { id } = query;
+    if (!id) {
+      return;
+    }
```

Comment:
Do you think throwing an error would be better here if id is required in the query?

Reply:
Good point, will update.